modules/monitoring-exposing-custom-application-metrics-for-horizontal-pod-autoscaling.adoc

```diff
@@ -11,11 +11,12 @@ You can use the `prometheus-adapter` resource to expose custom application metri
 
 .Prerequisites
 
-* You have a custom Prometheus instance installed. In this example, it is presumed that Prometheus was installed in a user-defined `custom-prometheus` project.
+* You have a custom Prometheus instance installed as a Prometheus pod managed by a deployment or statefulset but not by a Prometheus custom resource (CR).  
```

Comment:
Should it be stateful set as two separate words?

Reply:
No, but it should be `StatefulSet`. Good catch! Fixed.

https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/